src/schema-parser/asyncapi-schema-parser.ts

```diff
@@ -8,12 +15,63 @@ export function AsyncAPISchemaParser(): SchemaParser {
   }
 }
 
-function validate() {
+async function validate(input: ValidateSchemaInput<unknown, unknown>): Promise<SchemaValidateError[]> {
+  const schema = input.data as JSONSchema7;
+  let errors: SchemaValidateError[] = [];
+
+  try {
+    ajv.compile(schema);
+  } catch (error: any) {
+    if (error! instanceof Error) {
+      errors = ajvToSpectralErrors(error);
+    } else {
+      // Unknown and unexpected error
+      throw error;
+    }
+  }
```

Comment:
I have thought it over again and it is not a good solution, from a simple fact. Here we validate the scheme using the normal draft 07 Schema, but in our Schema Object we have 3 additional fields that do not exist in the JSON Schema - https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#fixed-fields-21, so I recommend switching to the solution from our current parser where we validate the scheme using the schema stored in our json schemas, like here https://github.com/asyncapi/parser-js/blob/master/lib/asyncapiSchemaFormatParser.js#L68

Reply:
@smoya Do you need help with that? :)

Reply:
No, was reading code so I could better understand things. That's why I wrote https://github.com/asyncapi/parser-js/issues/481#issuecomment-1195965993.

Btw, the code of this PR has been updated with what I had until then. feel free to review it again 🙏

Reply:
Done.